rocketchat-common/src/main/java/com/rocketchat/common/data/lightstream/document/ClientVersionsDocument.java

```diff
@@ -0,0 +1,50 @@
+package com.rocketchat.common.data.lightstream.document;
+
+import org.json.JSONObject;
+
+public class ClientVersionsDocument {
```

Comment:
We would like to keep all models as @AutoValue, it is easier to maintain, immutable, supports @NonNull and @Nullable annotaitons that helps integrating with Kotlin on the Android Client.

No need to change it now, but add a `// TODO - convert to AutoValue